### ISSUE TITLE
full.aln or .aln for newick/tree building

### ISSUE THREAD

**Issue by peflanag** [tseemann/snippy#401]
Hi,

I am trying to run snippy-core on a couple of Staph sequences. I have the sequences grouped by ST and ran snippy on them all and now I am running snippy-core. The only issue is whit some, the SNPs=0.

I know I have been told before to remove them and run again cause SNPs aren't called with them but how do i represent this on a tree? Is there a way to force snippy to lave them there so they are also on a tree? otherwise how to I represent the fact that there is no SNPs and they are identical? Should I use the full.aln file to make a newick rather than the .aln file?

I'm kind of at a loss and maybe I am not asking the right question! Any help would be great, cheers!

<img width="711" alt="Screenshot 2020-06-15 at 14 02 15" src="https://user-images.githubusercontent.com/32863969/84662636-fd789900-af13-11ea-9985-98cf6fbf5c53.png">

**Comment by tseemann** [comment]
Yes you can just give it core.full.aln - It will still "work" because the tree software will model the missing bases and Ns. 

But that's not your problem here.

Your problem is that E6305 and E6341.
2.4 Mbp of them are `unaligned`. That means no reads. Those samples don't have any Staph reads in them OR you didn't trim adaptors from those reads. You can try `--maxsoft 100` to try and save those samples.

**Comment by peflanag** [comment]
Hey @tseemann thanks for that. I didn't realise those samples were an issue. FastP must not have worked on them or I must have put the raw reads rather than the fastp output for them in the snippy directory. I will have a look at them again.

Cheers for the reply,

Peter

**Comment by peflanag** [comment]
hey @tseemann  just out of curiosity, would it have anything to do with the fact that some samples where --se  .fasta files? You see I had some samples and we didnt know the ST type and so I did de novo assmnly with SPAdes and then used the mlst command line program to dertermine the ST. Then I put all samples in the correct ST catagory and ran snippy using a scrip I looped for --R1 and --R2 .fq.gz as well as --se .fasta.

But after you pointed out the unaligned ones I used the --R1 and --R2 for those sampels and all worked perfectly! Is there something about the two formats? Or is it just by chance?

Cheers again!

P

![Screenshot 2020-06-16 at 09 20 54](https://user-images.githubusercontent.com/32863969/84750058-b7264700-afb2-11ea-88fb-e152d6ad2d32.png)
